pom.xml

```diff
@@ -23,7 +23,7 @@
 
 		<project-version>0.1.3</project-version>
 
-		<client-api.version>${project-version}</client-api.version>
+		<client-api.version>${project-version}-SNAPSHOT</client-api.version>
```

Comment:
Changing the `project-version` will change the version of all the artifacts. I'll make the necessary changed